src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    {person.age && (<p className="Person__age">{`I am ${person.age}`}</p>)}
+
+    {!person.isMarried && (<p className="Person__partner">I am not married</p>)}
+
+    {person.isMarried && (person.sex === 'm'
+      ? (
+        <p className="Person__partner">
+          {person.partnerName}
+          {' '}
+          is my wife
+        </p>
+      )
+      : (
+        <p className="Person__partner">
+          {person.partnerName}
+          {' '}
+          is my husband
+        </p>
+      )
+    )}
```

Comment:
Looks hard to understand and debug. Better prepare data before rendering content. You have a lot of repeated code - follow the DRY principle.